security-hub.tf

```diff
@@ -1,15 +1,13 @@
 module "monitoring_security_hub" {
-  source  = "dasmeta/monitoring/aws//modules/aws-security-hub-opsgenie"
-  version = "1.5.2"
+  source  = "dasmeta/monitoring/aws//modules/security-hub"
+  version = "1.19.5"
 
+  name = "account-security-hub-bridge"
```

Comment:
can we have this as variable in:
var.alarm_actions.security_hub_alarms variable with default set?

you removed usage of following variables but they still exist in variables.tf:
var.alarm_actions.security_hub_alarms.securityhub_action_target_name
var.alarm_actions.security_hub_alarms.sns_topic_name
var.alarm_actions.security_hub_alarms.protocol
var.alarm_actions.security_hub_alarms.link_mode